### PR TITLE
Fix CNI reload

### PR DESCRIPTION
**Pull request by madhanrm**
Fix the below issue
018-11-08 18:29:57: [Info] Run-Pod: crictl runp C:\scripts\pod-config-wcow.json
2018-11-08 18:29:57: [Info] Run-Pod: Pod create output = time="2018-11-08T18:29:57-08:00" level=fatal msg="run pod sandbox failed: rpc error: code = Unknown desc = failed to setup networking for sandbox \"51c8d25bcf21f588af68b878ca8e4b0ae8f80e3a98ecb0de46f8843d58da534e\": failed to setup network for sandbox \"51c8d25bcf21f588af68b878ca8e4b0ae8f80e3a98ecb0de46f8843d58da534e\": failed to find plugin \"loopback\" in path [C:\\crictl\\cni]"

### CODE REVIEW

**Comment by madhanrm** [comment]
@jterry75 @rjaini

File: pkg/server/update_runtime_config.go [hunk 1]
```diff
@@ -49,10 +50,21 @@ func (c *criService) UpdateRuntimeConfig(ctx context.Context, r *runtime.UpdateR
...
 		return &runtime.UpdateRuntimeConfigResponse{}, nil
 	}
+	var copts []cni.CNIOpt
+	if osruntime.GOOS != "windows" {
+		copts = append(copts, cni.WithLoNetwork)
+	}
+	copts = append(copts, cni.WithDefaultConf)
+
+	// Load the latest cni configuration to be in sync with the latest network configuration
+	if err := c.netPlugin.Load(copts...); err != nil {
```
Comment (jterry75) [inline pkg/server/update_runtime_config.go]: You dont want to call Load here right? You just want to make the opts and then wait for the "else if" to trigger the load?

File: pkg/server/status.go [hunk 2]
```diff
@@ -43,10 +44,20 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
...
+		copts = append(copts, cni.WithLoNetwork)
+	}
+	copts = append(copts, cni.WithDefaultConf)
 	// Load the latest cni configuration to be in sync with the latest network configuration
-	if err := c.netPlugin.Load(cni.WithLoNetwork, cni.WithDefaultConf); err != nil {
+	if err := c.netPlugin.Load(copts...); err != nil {
 		logrus.WithError(err).Errorf("Failed to load cni configuration")
 	}
+
+	// Load the latest cni configuration to be in sync with the latest network configuration
```
Comment (jterry75) [inline pkg/server/status.go]: Remove. This is a good fix to unify the above opts.

File: pkg/server/update_runtime_config.go (hunk 1 above)
Comment (madhanrm) [reply pkg/server/update_runtime_config.go]: Yes, Let me remove it

**Pull request merged**